src/ErrorHandling.h

```diff
@@ -169,6 +169,8 @@ typedef enum
     SUBCODE_CONFIGURATION_FAIL_MP_NICDATA1,
     SUBCODE_FAIL_MP_NICDATA_INIT1,
     SUBCODE_FAIL_INVALID_BASE_TRACK_MOTION_TYPE,
+    SUBCODE_FAIL_ADD_SERVICE_GET_ACTIVE_ALARM_INFO,
+    SUBCODE_FAIL_INIT_SERVICE_GET_ACTIVE_ALARM_INFO,
```

Comment:
I just realized that we also need to document the alarm codes.

Reply:
Of course, we need to wary of conflicts in the enum with #279

Reply:
Similar to #279: would the catch-all documentation suffice here?

Reply:
Yeah. There's nothing the user can do about this.